packages/framework/src/Model/Product/Search/ProductElasticsearchRepository.php

```diff
@@ -179,6 +190,15 @@ protected function extractIds(array $result): array
         return array_column($hits, '_id');
     }
 
+    /**
+     * @param array $result
+     * @return array
+     */
+    protected function extractHits(array $result): array
```

Comment:
Hits is term used by Elastic and could be confusing further in code. For example in `ProductOnCurrentDomainElasticFacade` is `return new PaginationResult($page, $limit, $productsResult->getTotal(), $productsResult->getHits());`. 
I would be much happier if it was named `extractResults` or `extractProducts` and in `ProductResult` accordingly

Reply:
I removed it from read model but this is repository is working with elastic so it shouldn't be a problem to use its namespace